smrf/utils/pycompat.py

```diff
@@ -36,10 +36,9 @@ def itervalues(d):
     pyreduce = reduce
     import __builtin__ as builtins
     from ConfigParser import SafeConfigParser
+
+
 try:
-    from cyordereddict import OrderedDict
-except ImportError:  # pragma: no cover
-    try:
-        from collections import OrderedDict
-    except ImportError:
-        from ordereddict import OrderedDict
+    from collections import OrderedDict
+except ImportError:
+    from ordereddict import OrderedDict
```

Comment:
Is there still Python 2 support in SMRF?

Reply:
No, but there are some leftovers from when there was. I only did this since VScode debugger was reporting an error every time. This should be cleaned up to remove the compatibility.